astroplan/__init__.py

```diff
@@ -16,3 +16,6 @@
 
     # Import everything in astroplan.core
     from .core import *
+
+    # Import everything in astroplan.exceptions
```

Comment:
Could you please remove this comment and the similar one above that was there before?
(It just repeats what the code says (and is not 100% accurate as-is))